src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -149,6 +166,8 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
       // the last icon does not need margin at the end
       sx={{ mr: index === platforms.length - 1 ? 0 : spacing }}
       platform={platform}
+      title={tooltips ? tooltips[index] : undefined}
+      placement={placements ? placements[index] : undefined}
```

Comment:
I think I would go with objects instead of indexes. Eg. `tooltips[platform]` and `placements[platform]`

Reply:
I agree, see my above comments which would solve this.